src/core/checkedint.d

```diff
@@ -682,10 +682,27 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 && x && (r / x) != y)
```

Comment:
separating this into 3 lines:
```
if (y >> 32 &&
x &&
(r/x)!=y)
```
will enable code coverage testing to check each line. I think thoroughly testing these case is particularly important for this function.